specificatie/resolved/openapi.yaml

```diff
@@ -7,7 +7,7 @@ info:
     Gegevens die er niet zijn of niet actueel zijn krijg je niet terug. Had een persoon bijvoorbeeld een verblijfstitel die nu niet meer geldig is, dan wordt die verblijfstitel niet opgenomen. In partners wordt alleen de actuele of de laatst ontbonden partner geleverd.
 
     Zie de [Features overzicht](https://brp-api.github.io/Haal-Centraal-BRP-bevragen/v2/features-overzicht) en [Getting started](https://brp-api.github.io/Haal-Centraal-BRP-bevragen/v2/getting-started) voor nadere toelichting.
-  version: 2.3.0
+  version: 2.4.1
```

Comment:
waarom wordt de nieuwe versie 2.4.1? Waar komt die patch versie vandaan?

De API zit nu op 2.4.0 (dat is de release met gezag niet te bepalen toelichting), specs zaten op 2.3.0 (software versie en specificatie versie lopen uit elkaar). Dan wordt de nieuwe softwareversie 2.5.0. Is het niet handig de specificatieversie gelijk te maken met het versienummer - minor versie - zoals we die ook in de releasenotes zetten? 
(patchversies van de software wijzigen zonder wijziging in de interface specificaties)

Reply:
edit: base branch aangepast naar api-specificaties-gezag-niet-te-bepalen

Dat komt omdat (voorlopige) versie 2.4.0 de toelichting bij GezagNietTeBepalen bevat en deze aanpassingen zijn daarop gebaseerd maar zijn later ontwikkeld. We kunnen deze aanpassingen eventueel in een nieuwe minor versie releasen of eventueel opnemen in de releasenotes van 2.4.0.

Reply:
PR #1831 (toelichting gezag niet te bepalen) maakt versie 2.4.0. Ik zei het hierboven dus verkeerd. 2.3.0 is nog oude(re) versie. Dus toevoegen toelichting tijdelijk geen gezag moet m.i. worden v2.5.0.